.github/ISSUE_TEMPLATE/2-wiki-migration.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: "🗃️ Wiki migration"
+labels: documentation
+about: Migrating Jenkins Wiki pages to jenkins.io
+---
+
+* Page to migrate: `TODO`
+* Destination on jenkins.io: `TODO`
+
+## Migration notes
+
+<!-- If there are any considerations to keep in mind during the migration, please add them here -->
+
+* N/A
+
+## For contributors
+
+* [Wiki Migration Guidelines](https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#moving-documentation-from-jenkins-wiki)
```

Comment:
@MarkEWaite https://issues.jenkins-ci.org/browse/INFRA-2328 contains slightly different guidelines. Could we merge them?

Reply:
Yes, I'll submit a pull request to refine the guidelines in the contributing file and replace them in [INFRA-2328](https://issues.jenkins-ci.org/browse/INFRA-2328) with a link.